doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -116,16 +116,21 @@ Table 2:<a name="table-supplier-outcomes"></a> Proposed Meaning for Supplier Pri
 
 ### Deploying Patches
 
-Whether or not to deploy available remediation is a binary decision. However, additional defensive mitigations may be necessary sooner than patches are available and may be advisable after patches are applied. We recognize that vulnerability management actions are different when a patch is available and when it is not available.
+Whether or not to deploy available remediation is a binary decision because remediation, when successful, eliminates a vulnerability in a specific system.
+However, additional defensive mitigations may be necessary sooner than patches are available and may be advisable after patches are applied.
+Mitigation, from a tactical perspective, is an attempt to block or minimize risk to a vulnerable system without actually eliminating the vulnerability.
+We recognize that vulnerability management actions are different when a patch is available and when it is not available.
```

Comment:
Should this replace patch etc with remediation for consistency?

Reply:
https://github.com/CERTCC/SSVC/pull/128/commits/4b2d110e101a9c6e0138c1fc1066e8a2316626c7